src/CmsInlineFormAction.php

```diff
@@ -103,6 +117,9 @@ public function FieldHolder($properties = array())
         if ($this->readonly) {
             $attrs .= ' style="display:none"';
         }
+        if (\strlen($this->submitSelector)) {
```

Comment:
Wrong project settings my bad!
```suggestion
        if (strlen($this->submitSelector)) {
```